app/models/step_by_step_page.rb

```diff
@@ -128,6 +128,10 @@ def links_last_checked_date
     steps.map(&:links_last_checked_date).reject(&:blank?).max
   end
 
+  def should_check_links?
```

Comment:
Should this also return false if there are no links to check? E.g if there aren't any `navigation_rules`?